test/run_sbm_gwf.jl

```diff
@@ -13,7 +13,7 @@ flush(model.writer.csv_io)  # ensure the buffer is written fully to disk
     row = csv_first_row(model.writer.csv_path)
 
     @test row.time == DateTime("2000-06-01T00:00:00")
-    @test row.Q_av ≈ 0.01620324716944374f0
+    @test row.Q_av ≈ 0.01619703129434486f0
```

Comment:
I don't directly see what causes these changes, do you know why this happened? Same for the other differences in Q in the other test.

Reply:
Ah I see in the commit history, it is probably related to the new `TimeStepping`, correct?